packages/round-manager/src/features/program/ViewProgramPage.tsx

```diff
@@ -63,7 +63,7 @@ export default function ViewProgram() {
           >
             <div className="flex-1 min-w-0">
               <p className="text-sm pb-3 mb-1 font-medium text-gray-900">
-                {round.roundMetadata!.name}
+                {round.roundMetadata?.name ?? ''}
```

Comment:
Hmm i'm conflicted on this.
Ideally the metadata will always have a name so in that sense this shouldn't be optional right ? 

Does this error out if we we `{round.roundMetadata.name''}` @bhargavaparoksham  ?

cc @shavinac

Reply:
Yes, it is throwing an error, if we {round.roundMetadata.name''}

Reply:
i think the `Round` type has `roundMetadata` set as an optional field, since the round contract could have a null/empty metadata pointer. But `name` is a required field on `roundMetadata`

in that case, we could add an initial filtering step to remove any rounds without metadata before generating the round list UI, like `rounds.filter(round => !!round.roundMetadata).map(...)`. then the ui should be able to show `<p>{round.roundMetadata.name}</p>` without using non-null assertion or causing TS compiler errors

Reply:
> rounds.filter(round => !!round.roundMetadata).map(...)

Yup i like this 🙌🏼